docs/source/learning.md

```diff
@@ -52,29 +65,52 @@ PyMC's own {doc}`glossary` defines many core terms and provides useful reference
 
 ## Using PyMC
 
-(the path in pymc-examples repo excluding the `examples` folder, or using manual anchors if we decide to create them).
+### {octicon}`code-square;1em;sd-text-info` Example notebooks
+The {doc}`nb:index` hosts a collection of over 90 Jupyter notebooks using PyMC.
+They range from notebooks addressed to people who are new to PyMC to notebooks
+addressed to expert Bayesian practitioners that use PyMC.
 
-::::{grid} 2
-:gutter: 4
+The collection is organized using tags and categories to make it easy to explore
+and focus on the examples that match your interests. You can also
+check the {ref}`nb:object_index` to find examples using a specific
+PyMC function or class.
 
-:::{grid-item-card} TODO
+### {octicon}`repo;1em;sd-text-info` Intermediate books
+There are several great books available to deepen your understanding
+of Bayesian statistics. Many of them also have their code
+examples translated to PyMC and publicly available at
+the [pymc-resources](https://github.com/pymc-devs/pymc-resources) repository.
 
+In this section we highlight one book that uses PyMC and TensorFlow
+Probability written by members of the PyMC team.
+For the complete list see the {ref}`books` page.
 
-TODO
-:::
+#### Bayesian Modeling and Computation in Python
+
+By Osvaldo Martin, Ravin Kumar and Junpeng Lao
+
+`<add description here>`
```

Comment:
@canyon289 @aloctavodia @junpenglao do you want to add a paragraph here similar to https://docs.pymc.io/en/latest/learning.html#bayesian-methods-for-hackers and https://docs.pymc.io/en/latest/learning.html#bayesian-analysis-with-python ones?